public/app/features/migrate-to-cloud/onprem/useNotifyOnSuccess.tsx

```diff
@@ -45,6 +45,8 @@ function getTranslatedMessage(snapshot: GetSnapshotResponseDto) {
       types.push(t('migrate-to-cloud.migrated-counts.datasources', 'data sources'));
     } else if (type === 'FOLDER') {
       types.push(t('migrate-to-cloud.migrated-counts.folders', 'folders'));
+    } else if (type === 'LIBRARY_ELEMENT') {
+      types.push(t('migrate-to-cloud.migrated-counts.library_elements', 'library elements'));
```

Comment:
Nothing wrong with this, but for the next type I would consider just changing the message to `Successfully migrated 5 items to your Grafana Cloud instance`

Reply:
Agree! We're going to be doubling the resources next time around with Alerts.